recipes/shtns/bld.bat

```diff
@@ -0,0 +1,16 @@
+REM set TMP="%LOCALAPPDATA%\Temp"
+REM set TEMP="%LOCALAPPDATA%\Temp"
+REM set TMPDIR="%LOCALAPPDATA%\Temp"
+if "%ARCH%" == "64" (
+set GCC_ARCH=x86_64-w64-mingw32
+set EXTRA_FLAGS="-DMS_WIN64 -I/mingw-w64/%GCC_ARCH%/include/"
+) else (
+set GCC_ARCH=i686-w64-mingw32
+set EXTRA_FLAGS="-I/mingw-w64/%GCC_ARCH%/include/"
+)
+bash -lc "mkdir -p /tmp; ln -s ${LOCALAPPDATA}/Temp /tmp"
+bash -lc "echo $CYGWIN_PREFIX"
+bash -lc "export LDFLAGS=-L`cygpath -u $LIBRARY_LIB`; echo $LDFLAGS; ls -l `cygpath -u $LIBRARY_LIB`"
+bash -lc "cd src; export LDFLAGS=-L`cygpath -u $LIBRARY_LIB`; export PYTHON=`which python`; ./configure --disable-openmp --enable-python --build=$GCC_ARCH --host=$GCC_ARCH --target=$GCC_ARCH --prefix=$CYGWIN_PREFIX CFLAGS=$EXTRA_FLAGS"
+bash -lc "cd src; export CFLAGS=$EXTRA_FLAGS; make"
+bash -lc "cd src; make install"
```

Comment:
Can you make an install script for this.  Also each of these commands needs an exit condition check

Reply:
Thanks very much for taking a look, and the helpful comments!

As hard as I tried, getting an autotools build with conda working properaly on windows with the mixing of compilers/linkers was just not working. As such for now I've just made this not support windows. As far as I can find, there aren't even any build instructions for windows on the original author's repository/codebase so I think that's an acceptable outcome.